blueflood-core/src/main/java/com/rackspacecloud/blueflood/service/ShardStateManager.java

```diff
@@ -47,7 +47,7 @@
     private static final Map<Granularity, Meter> granToReRollMeters = new HashMap<Granularity, Meter>();
     private static final Map<Granularity, Meter> granToDelayedMetricsMeter = new HashMap<Granularity, Meter>();
 
-    private static final long DELAYED_METRICS_MAX_ALLOWED_DELAY = Configuration.getInstance().getLongProperty( CoreConfig.BEFORE_CURRENT_COLLECTIONTIME_MS );
+    public static final long DELAYED_METRICS_MAX_ALLOWED_DELAY = Configuration.getInstance().getLongProperty( CoreConfig.BEFORE_CURRENT_COLLECTIONTIME_MS );
```

Comment:
Since this variable is similar to DELAYED_METRICS_ROLLUP_DELAY_MILLIS could we rename it?

one half-baked idea:
ASSUMING_REROLL_WITHIN_THIS_TIMESPAN (that's what it is, an assumption)